### DISCUSSION TITLE
Dynamically allocate node

### DISCUSSION THREAD

**Discussion by VjDharnish** [milvus-io/milvus#28230] [Q&A and General discussion]
How milvus dynamically allocates nodes. if I need to disable dynamic allocation, then how to do it. Need explanation with respect to nodes configuration

**Comment by yhmo** [comment]
Not sure what do you mean "dynamically allocates nodes".
If you mean "auto-scaling", this discussion gives the answer: https://github.com/milvus-io/milvus/discussions/25422
